gax-grpc/src/main/java/com/google/longrunning/OperationsClient.java

```diff
@@ -519,6 +519,61 @@ private final void deleteOperation(DeleteOperationRequest request) {
     return stub.deleteOperationCallable();
   }
 
+  /**
+   * Waits for the specified long-running operation until it is done or reaches at most a specified
```

Comment:
"Waits for the specified long-running operation until it is" --> "Waits until the specified long-running operation is"

Reply:
This is meant to reflect the [RPC's comment](https://github.com/googleapis/googleapis/blob/master/google/longrunning/operations.proto#L113) verbatim. If I change it here, we'll need to update the protos as well.

Reply:
Is this PR checking in generated code? I'm not sure we need to reflect the RPC's comment verbatim (though, yes, it should be changed there too.)

Reply:
The delta here is nearly identical to the generated code, so yes. I'm adding it in this way (as opposed to copy-pasting the new GAPIC client entirely) to preserve existing manual changes and ensure that dependencies won't break.

Updated the comment.